docs/antora.yml

```diff
@@ -0,0 +1,6 @@
+name: android
+title: The Android App
```

Comment:
Maybe `Android Application`?

Reply:
Not sure, @tboerger. Would it make much of a difference eitherway?

Reply:
Just thinking about how this looks like on the docs ;)

Reply:
The current docs use this naming style, so I'm leaving it as is.